packages/mc-scripts/src/commands/config:sync.js

```diff
@@ -0,0 +1,136 @@
+const omit = require('lodash/omit');
+const prompts = require('prompts');
+const chalk = require('chalk');
+const { processConfig } = require('@commercetools-frontend/application-config');
+const CredentialsStorage = require('../utils/credentials-storage');
+const {
+  fetchCustomApplication,
+  updateCustomApplication,
+  createCustomApplication,
+  fetchUserOrganizations,
+} = require('../utils/graphql-requests');
+const updateApplicationIdInCustomApplicationConfig = require('../utils/update-application-id-in-custom-application-config');
+
+const credentialsStorage = new CredentialsStorage();
+
+const getMcUrlLink = (mcApiUrl, organizationId, applicationId) => {
+  const mcUrl = mcApiUrl.replace('mc-api', 'mc');
+  const customAppLink = `${mcUrl}/account/organizations/${organizationId}/custom-applications/owned/${applicationId}`;
+  return customAppLink;
+};
+
+const configSync = async () => {
+  const applicationConfig = processConfig();
+  const { data: localCustomAppData } = applicationConfig;
+  const { mcApiUrl } = applicationConfig.env;
+
+  if (!credentialsStorage.isSessionValid(mcApiUrl)) {
+    throw new Error(
+      `You don't have a valid session for the ${mcApiUrl} environment. Please, run the "mc-scripts login" command to authenticate yourself.`
+    );
+  }
+
+  const token = credentialsStorage.getToken(mcApiUrl);
+  const fetchedCustomApplication = await fetchCustomApplication({
+    mcApiUrl,
+    token,
+    entryPointUriPath: localCustomAppData.entryPointUriPath,
+  });
+
+  if (!fetchedCustomApplication) {
+    const userOrganizations = await fetchUserOrganizations({ mcApiUrl, token });
+
+    let organizationId, organizationName;
+    if (userOrganizations.total > 1) {
+      const organizationChoices = userOrganizations.results.map(
+        (organization) => ({
+          title: organization.name,
+          value: organization.id,
+        })
+      );
+
+      organizationId = await prompts({
+        type: 'select',
+        name: 'organizationId',
+        message: 'Select Organization',
+        choices: organizationChoices,
+        initial: 0,
+      })['organizationId'];
+
+      organizationName = organizationChoices.find(
+        ({ value }) => value === organizationId
+      ).title;
+    } else {
+      organizationId = userOrganizations.results[0].id;
+      organizationName = userOrganizations.results[0].name;
+    }
```

Comment:
I would also check if there are no organizations. In this case we need to throw an error.

And maybe invert the condition.
```suggestion
    if (userOrganizations.total === 0) {
      throw new Error(`It seems you are not an admin of any Organization. Please make sure to be part of the Administrators team of the Organization you want the Custom Application to be configured to.`);
    }
    if (userOrganizations.total === 1) {
      organizationId = userOrganizations.results[0].id;
      organizationName = userOrganizations.results[0].name;
    } else {
      const organizationChoices = userOrganizations.results.map(
        (organization) => ({
          title: organization.name,
          value: organization.id,
        })
      );

      organizationId = await prompts({
        type: 'select',
        name: 'organizationId',
        message: 'Select Organization',
        choices: organizationChoices,
        initial: 0,
      })['organizationId'];

      organizationName = organizationChoices.find(
        ({ value }) => value === organizationId
      ).title;
    }
```